webpack/src/Components/Environments/Welcome.js

```diff
@@ -14,19 +14,25 @@ export const WelcomeEnv = ({ canCreate }) => {
   const description = (
     <>
       <FormattedMessage
-        id="puppetenv-welcome"
+        id="puppetenv-welcome1"
         defaultMessage={__(
-          'If you are planning to use Foreman as an external node classifier you should provide information about one or more environments.{newLine}This information is commonly imported from a pre-existing Puppet configuration by the use of the {puppetClasses}.'
+          'If you are planning to use Foreman as an external node classifier you should provide information about one or more environments.'
+        )}
+      />
+      <br />
+      <FormattedMessage
+        id="puppetenv-welcome2"
+        defaultMessage={__(
+          'This information is commonly imported from a pre-existing Puppet configuration by the use of the {puppetClasses} and environment importer.'
```

Comment:
What I was suggesting earlier:
```suggestion
          'This information is commonly imported from a pre-existing Puppet configuration by the use of the {link_start}Puppet classes{link_end} and environment importer.'
```

That's what I meant with building up sentences or stringing sentences together. Splitting up the first part (with the br) into a separate message isn't the problem I was talking about.

Reply:
I think I understand what you mean, I did not get this to work, though. You sure that this is supported? Not sure if you can pass complete React elements/HTML to the values place holders. Do you have a working example?

Reply:
I think then this can be done like this? 
and disable eslint long line rule in the file or something
```js
{__('If you are planning to use Foreman as an external node classifier you should provide information about one or more environments.')}
<br />
<FormattedMessage
        id="puppetenv-welcome2"
        defaultMessage={__('This information is commonly imported from a pre-existing Puppet configuration by the use of the {puppetClasses} and environment importer.'/>
```

Reply:
> I think then this can be done like this? and disable eslint long line rule in the file or something
> 
> ```js
> {__('If you are planning to use Foreman as an external node classifier you should provide information about one or more environments.')}
> <br />
> <FormattedMessage
>         id="puppetenv-welcome2"
>         defaultMessage={__('This information is commonly imported from a pre-existing Puppet configuration by the use of the {puppetClasses} and environment importer.'/>
> ```

Looks like the easiest. @MariaAga Do you know if one can split up HTML tags across multiple value placeholders such as @ekohl suggested https://github.com/theforeman/foreman_puppet/pull/334#discussion_r1109632141?

Reply:
Because a full sentence gives the translators a better context of what they're translating.

Reply:
Then should we just rename the variable name in the string for the translator? like `{puppetClassesLinkToDocs}`

Reply:
I tried to update the PR based on this discussion. @ekohl Unfortunately, I did not get the rake task to for the translations to run. Do you mind verifying if it works now?